extensions/amp-analytics/amp-analytics.md

```diff
@@ -476,6 +476,13 @@ In addition, all of the variables supported by
 [AMP HTML Substitutions Guide](../../spec/amp-var-substitutions.md)
 are also supported.
 
+Variables are resolved asynchronously and can delay the request until they are
```

Comment:
are or can?

I'd assume some are resolved synchronously or close enough to sync to not be a factor.

Reply:
Yeah, some async ones may resolve quickly.